### PR TITLE
Conditional Media Contacts in Feature Story metadata

### PR DESCRIPTION
**Pull request by iamrentman**
- Adds check for empty Media Contacts field values so markup does not print if none are added
- Compiled CSS + JS
- Increment `metadata-fields` for deployment to Component Service

# READY FOR REVIEW

# Summary
- Media Contacts field markup was printing when non are added leaving a large space between it and the contributor rows above.

# Review By (Date)
- ASAP

# Criticality
- 9

# Review Tasks

## Setup tasks and/or behavior to test

1. Check out this branch
2. npm run build, npm start
3. Look at the [metadata-fields](http://localhost:3000/r/stanford-components/metadata-fields/0.0.24?_previewKey=default) component and make sure the space between the rows is consistent. I have removed the Media Contacts from the preview file to make this easier.

# Associated Issues and/or People
- [UCP-3177](https://stanfordits.atlassian.net/browse/UCP-3177)

[UCP-3177]: https://stanfordits.atlassian.net/browse/UCP-3177?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by hmalcolmSquiz** [comment]
Code looks good from my perspective - I think there might still be a console.log in packages/metadata-fields/FeaturedMetadata.jsx

**Manifest version**
For the manifest version, in develop we pushed everything to version `0.1.0`, if you bump this to `0.1.1` this would be a newer version. Alternatively, if you consider this as releasing a new feature for this component, you can also just go straight to `0.2.0`. 

**Conflicts**
I'd recommend merging develop into this feature branch and resolving these conflicts there first. Use your new version (from above) for the manifest number, run an `npm run build` to handle all the global/* files and I imagine that you'd want to keep both for `packages/__globalPreview/frontend.html`. That should take care of all of them and it would then be ready to merge and deploy.

File: packages/metadata-fields/FeaturedMetadata.jsx [hunk 1]
```diff
@@ -145,7 +145,7 @@ export default function FeaturedMetadata({ data }) {
         ""
       )}
       {otherRows.length > 0 ? (
-        <div className="su-flex su-w-full su-flex-col su-justify-center su-items-center su-mt-32 md:su-mt-61 md:su-flex-row md:su-gap-20 lg:su-gap-40">
+        <div className="su-flex su-w-full su-flex-col su-justify-center su-items-center su-rs-mt-4 md:su-flex-row md:su-gap-20 lg:su-gap-40">
```
Comment (yvonnetangsu) [inline packages/metadata-fields/FeaturedMetadata.jsx:148]: I updated this row to use the `su-rs-mt-4` spacer to match the below rows

File: components/metadata-fields/manifest.json [hunk 2]
```diff
@@ -1,7 +1,7 @@
 {
   "$schema": "http://localhost:3000/schemas/v1.json#",
   "name": "metadata-fields",
-  "version": "0.1.0",
+  "version": "0.1.1",
```
Comment (yvonnetangsu) [inline components/metadata-fields/manifest.json:4]: As suggested by @hmalcolmSquiz , I updated the version to 0.1.1

File: packages/__globalPreview/frontend.html [hunk 3]
```diff
@@ -39,10 +39,17 @@
...
+            data-hydration-props='{"type":"Featured","authors":[{"asset_name":"JB Christy","asset_assetid":"128503"}],"producers":[{"asset_name":"JB Christy","asset_assetid":"128503"}],"writers":[{"asset_name":"JB Christy","asset_assetid":"128503"}],"editors":[{"asset_name":"JB Christy","asset_assetid":"128503"}],"videographers":[],"photography":[],"media":[],"campus":{"asset_name":"Stanford BeWell","asset_assetid":"28337","asset_url":""},"related":[{"asset_assetid":"128103","asset_name":"Cardinal at Work","asset_url":"","target_id":""},{"asset_assetid":"28433","asset_name":"Campus beauty","asset_url":"https://sug-web.matrix.squiz.cloud/report/university-news/campus-and-facilities/campus-beauty","target_id":"125110"}]}'
```
Comment (yvonnetangsu) [inline packages/__globalPreview/frontend.html:52]: Not completely sure if we want to keep all this before merging - please let me know @broleomo 😄

**Comment by yvonnetangsu** [comment]
> Code looks good from my perspective - I think there might still be a console.log in packages/metadata-fields/FeaturedMetadata.jsx
> 
> **Manifest version** For the manifest version, in develop we pushed everything to version `0.1.0`, if you bump this to `0.1.1` this would be a newer version. Alternatively, if you consider this as releasing a new feature for this component, you can also just go straight to `0.2.0`.
> 
> **Conflicts** I'd recommend merging develop into this feature branch and resolving these conflicts there first. Use your new version (from above) for the manifest number, run an `npm run build` to handle all the global/* files and I imagine that you'd want to keep both for `packages/__globalPreview/frontend.html`. That should take care of all of them and it would then be ready to merge and deploy.

Thank you @hmalcolmSquiz ! I have merged in develop and resolved the merge conflicts, removed the console.log, did a few minor tweaks then increment the version to 0.1.1 and did npm run build again.

Since Justin is out, @broleomo if I could bother you to take a look at this, then perhaps we're ready to merge this into `develop`.

**Review by broleomo — APPROVED** [review]

**Pull request merged**